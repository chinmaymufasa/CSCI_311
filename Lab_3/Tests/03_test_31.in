2
34
5
2
44
5
2
4
5
2
13
5
2
126
5
2
170
5
2
82
5
2
84
5
2
130
5
2
104
5
2
185
5
2
60
5
2
99
5
2
56
5
2
127
5
2
168
5
2
184
5
2
91
5
2
118
5
2
142
5
1
20
-1