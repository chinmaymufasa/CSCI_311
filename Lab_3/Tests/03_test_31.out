34 
34 44 
4 34 44 
4 13 34 44 
4 13 34 44 126 
4 13 34 44 126 170 
4 13 34 44 82 126 170 
4 13 34 44 82 84 126 170 
4 13 34 44 82 84 126 130 170 
4 13 34 44 82 84 104 126 130 170 
4 13 34 44 82 84 104 126 130 170 185 
4 13 34 44 60 82 84 104 126 130 170 185 
4 13 34 44 60 82 84 99 104 126 130 170 185 
4 13 34 44 56 60 82 84 99 104 126 130 170 185 
4 13 34 44 56 60 82 84 99 104 126 127 130 170 185 
4 13 34 44 56 60 82 84 99 104 126 127 130 168 170 185 
4 13 34 44 56 60 82 84 99 104 126 127 130 168 170 184 185 
4 13 34 44 56 60 82 84 91 99 104 126 127 130 168 170 184 185 
4 13 34 44 56 60 82 84 91 99 104 118 126 127 130 168 170 184 185 
4 13 34 44 56 60 82 84 91 99 104 118 126 127 130 142 168 170 184 185 
Not found
